docs/source/backends/raspbian.rst

```diff
@@ -0,0 +1,683 @@
+
+General settings
+----------------
+
+The general settings reside in the ``general`` key of the
+*configuration dictionary*, which follows the
+`NetJSON General object <http://netjson.org/rfc.html#general1>`_ definition
+(see the link for the detailed specification).
+
+General settings example
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following *configuration dictionary*:
+
+.. code-block:: python
+
+    {
+        "general": {
+            "hostname": "routerA",
+            "timezone": "UTC",
+            "ula_prefix": "fd8e:f40a:6701::/48"
+        }
+    }
+
+Will be rendered as follows::
+
+    config: /etc/hostname
+    routerA
+
+    run commands:
+    $ timedatectl set-timezone UTC
+
+After modifying the config files run the following command to change the
+hostname::
+
+    $ /etc/init.d/hostname.sh start
+
+Network interfaces
+------------------
+
+The network interface settings reside in the ``interfaces`` key of the
+*configuration dictionary*, which must contain a list of
+`NetJSON interface objects <http://netjson.org/rfc.html#interfaces1>`_
+(see the link for the detailed specification).
+
+There are 3 main type of interfaces:
+
+* **network interfaces**: may be of type ``ethernet``, ``virtual``, ``loopback`` or ``other``
+* **wireless interfaces**: must be of type ``wireless``
+* **bridge interfaces**: must be of type ``bridge``
+
+Loopback Interface
+~~~~~~~~~~~~~~~~~~
+
+The following *configuration dictionary*:
+
+.. code-block:: python
+
+    {
+        "interfaces": [
+            {
+                "name": "lo",
+                "type": "loopback",
+                "addresses": [
+                    {
+                        "address": "127.0.0.1",
+                        "mask": 8,
+                        "proto": "static",
+                        "family": "ipv4"
+                    }
+                ]
+            }
+        ]
+    }
+
+Will be rendered as follows::
+
+    config: /etc/network/interfaces
+    auto lo
+    iface lo inet static
+        address 127.0.0.1
+        netmask 255.0.0.0
+
+Dualstack (IPv4 & IPv6)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The following *configuration dictionary*:
+
+.. code-block:: python
+
+    {
+        "interfaces": [
+            {
+                "name": "eth0",
+                "type": "ethernet",
+                "addresses": [
+                    {
+                        "family": "ipv4",
+                        "proto": "static",
+                        "address": "10.27.251.1",
+                        "mask": 24
+                    },
+                    {
+                        "family": "ipv6",
+                        "proto": "static",
+                        "address": "fdb4:5f35:e8fd::1",
+                        "mask": 48
+                    }
+                ]
+            }
+        ]
+    }
+
+Will be rendered as follows::
+
+    config: /etc/network/interfaces
+    auto eth0
+    iface eth0 inet static
```

Comment:
use 4 spaces indentation for consistency with other examples on the same page which are using 4